catalog-info.yaml

```diff
@@ -0,0 +1,21 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'crowdsourcehinter'
+  description: >
+            This provides a student with hints when they
+            incorrectly answer a problem within a course
+  links:
+    - url: "https://github.com/openedx/crowdsourcehinter"
+      title: "openedx/crowdsourcehinter"
+      icon: "GitHub"
```

Comment:
This link is not necessary, the repo is automatically linked in Backstage already.

Reply:
updated